edgeless_api/src/node_registration.rs

```diff
@@ -1,19 +1,57 @@
 // SPDX-FileCopyrightText: © 2024 Technical University of Munich, Chair of Connected Mobility
 // SPDX-License-Identifier: MIT
+
 #[derive(Debug, Clone, PartialEq)]
 pub struct ResourceProviderSpecification {
     pub provider_id: String,
     pub class_type: String,
     pub outputs: Vec<String>,
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub struct NodeCapabilities {
+    // Number of (actual or virtual) CPUs associated with the edge node.
+    pub num_cpus: u32,
+    // Name of the CPU model.
+    pub model_name_cpu: String,
```

Comment:
Is that meaningful? We don't care about marketing names during scheduling. As I see it, we care about uarc + features.

Reply:
Again, I just implemented the first bunch of fields from a document that has been around for _weeks_.

If a field does not make sense, let's discuss this _in the document_ and then we can happily remove it from the code, as well. Meanwhile, it should not hurt, right?

In any case: what is not useful for one person (or policy) may be used instead by another person (or policy), who knows? 🤷‍♂️